### ISSUE TITLE
Enabled unused warnings on Scala 2.12

### ISSUE THREAD

**Issue by armanbilge** [typelevel/sbt-typelevel#307]
Somehow these fell through the cracks. H/t @satorg

**Comment by satorg** [comment]
I believe this is the line that disables the unused warning in Scala 2.12:
https://github.com/typelevel/sbt-typelevel/blob/c810fa9f8d3d71c9fc18158537699ac7f62891e1/settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala#L88

I can file a PR that would remove it, unless there is a better way.

**Comment by armanbilge** [comment]
Yes, that's the quick fix and is probably ok.

The part that bothers me is that it was introduced in https://github.com/djspiewak/sbt-spiewak/commit/95a45d0ea91a015c1e38e9f56d5533bb3acd8f28, but somehow unused warnings are still working there. So it would be nice to have a deeper explanation of this, but probably not worth the effort.

**Comment by satorg** [comment]
Btw, shouldn't the issue's title be "Disabled unused warnings..."? Because they're currently disabled, actually.

**Comment by armanbilge** [comment]
It should be "enable", as is, enable it !!! 😆

**Comment by armanbilge** [comment]
> but somehow unused warnings are still working there

No, I think I'm wrong about that actually. Let's just fix it and move on.
